pkg/ccl/streamingccl/streamingest/testdata/data-0

```diff
@@ -0,0 +1,1000 @@
+0,A
```

Comment:
If you wanted, you could probably replace this with a small in-memory http server that generates csv content.